mlflow/tracing/display/display_handler.py

```diff
@@ -55,6 +53,7 @@ def _display_traces_post_run(self, result):
             if get_ipython() is None:
                 return
 
+            MAX_TRACES_TO_DISPLAY = MLFLOW_MAX_TRACES_TO_DISPLAY_IN_NOTEBOOK.get()
```

Comment:
should we truncate traces in `display_traces`?

Reply:
I think it's not necessary, since ultimately this is the function that displays the traces